files/system/usr/libexec/setup-luks-fido2-unlock

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+# Copyright 2024 Universal Blue
+#
+# This file includes code that is licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
is there still ublue code in this? if not we can remove this comment

Reply:
I copied their code surrounding verification that there is a luks disk, identifying it, etc.